index.html

```diff
@@ -618,6 +618,93 @@ <h4>
         </section>
       </section>
     </section>
+    <section id="automation" class="informative">
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document define [[WEBDRIVER2]]
+        <a href="https://w3c.github.io/webdriver/#dfn-extension-commands">
```

Comment:
You're not supposed to link directly to the concepts you use. Look at the rest of the document and at the ReSpec documentation -- you need to use right notation (e.g. `[=foo=]` or `{{Bar}}`) to refer to the concepts, and ReSpec will in turn add the right links, bibliographic entries and references for you.